tests/ui_tools/test_popups.py

```diff
@@ -185,6 +186,127 @@ def test_feature_level_content(self, mocker, zulip_version):
         )
 
 
+class TestFullMsgView:
+    @pytest.fixture(autouse=True)
+    def mock_external_classes(self, mocker, msg_box, initial_index):
+        self.controller = mocker.Mock()
+        mocker.patch.object(self.controller, 'maximum_popup_dimensions',
+                            return_value=(64, 64))
```

Comment:
This seems conveniently similar to the hardcoded "magic" value in the actual code; maybe we should parametrize over things somehow so a test notices, or at least fix that width issue!

Reply:
(See my other comment)